ha/util/conf_store.py

```diff
@@ -103,6 +143,8 @@ def set_cluster_cardinality(self, index):
         # Combine the lists data_pods, server_pod, control_pods and find unique machine ids
         watch_pods = data_pods + server_pods + control_pods
         watch_pods = list(set(watch_pods))
+        matched_node_labeled_dict = ConftStoreSearch.get_matched_labels(index, watch_pods)
+        node_id_name_mapping_dict = ConftStoreSearch.get_node_name_to_machine_id_mapping(index, watch_pods)
```

Comment:
if pods are going to have only machine-ids, this dict will be empty ?

Reply:
Tanuja confirmed and replied that, irrespective of the labels, machine-id and pod_name will always be present.
Please check her above comment.